packages/falter-berlin-tunneldigger/files/tunneldigger.hotplug

```diff
@@ -24,13 +24,34 @@ interface_disabled() {
         echo "${disabled}"
 }
 
+srv_lookup() {
+        local srv="$1"
+        local i
+        local srvlist=""
+        for i in $srv; do
+                local lookup=$(nslookup -q=SRV $i | grep $i | cut -d "=" -f 2)
+                local OLDIFS=$IFS
+                IFS=$'\n'
+                local j
+                for j in $lookup; do
+                        local host=$(echo $j | cut -d ' ' -f 5)
+                        local port=$(echo $j | cut -d ' ' -f 4)
+                        append srvlist "${host}:${port}"
+                done
+                IFS=$OLDIFS
+        done
+
+        echo "${srvlist}"
+}
+
```

Comment:
This function is defined twice. Here and in the init-script. Please consider to move it into a library-file.

Reply:
I have the feeling that in the future we will just get rid of the init script.  So, I would prefer to leave it the way it is.

Reply:
okay, sounds good.